packages/vc-verification/package.json

```diff
@@ -59,8 +57,10 @@
     "ganache-cli": "^6.12.2",
     "ipfs": "^0.58.3",
     "ipfsd-ctl": "3.0.0",
+    "ipfs-http-client": "^43.0.0",
     "js-sha3": "^0.8.0",
     "jsonwebtoken": "^9.0.0",
+    "lodash": "^4.17.21",
```

Comment:
`lodash` is used here: https://github.com/energywebfoundation/ew-credentials/blob/a1e6ed6129485c2a4f33db4963ddb30684d144c2/packages/vc-verification/src/utils/formatters.ts
Is this doesn't seem to be only for tests?